src/payloads/message.ts

```diff
@@ -96,6 +133,11 @@ export const getCategory = (
   const senderService = services.find(
     s => s.service_id === message.sender_service_id
   )!;
+  if ((message as LegalMessageWithContent).legal_message) {
```

Comment:
```suggestion
  if (LegalMessageWithContent.decode(message).isRight()) {
```
pretty the same but using the dedicated codec